test/utils/mocks/CLI.js

```diff
@@ -2,6 +2,7 @@ const program = require('../../../bin');
 
 const cli = (args, callback) => {
     const argv = args.split(' ');
+    argv.push('-t');
```

Comment:
Do we still need this?

Reply:
Yes, until we find something with ```exec``` we need this :(